tests/utils/test_file_lookups.py

```diff
@@ -50,58 +49,51 @@ def test_project_id_exception(mock_gitlab):
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_project_id_success(mock_gitlab):
     """Check resolving GitLab Project ID is successful."""
-    project_id = 30
-
-    mock_gitlab.Gitlab.return_value.getproject.return_value = {'id': project_id}
-
+    project_id = object
+    mock_gitlab.Gitlab.return_value.projects.get.return_value = object
     seeker = FileLookup()
-
-    assert seeker.project_id == project_id
+    assert seeker.project == project_id
 
 
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_get(gitlab):
     """Check _get_ method."""
     my_git = FileLookup()
 
-    my_git.server.getfile.return_value = {'content': base64.b64encode(TEST_JSON_BYTES)}
-
-    result = my_git.get()
-    assert isinstance(result, str)
-    assert TEST_JSON == my_git.get()
+    with mock.patch.object(FileLookup, 'remote_file', return_value=TEST_JSON) as mock_method:
+        result = my_git.get()
+        assert isinstance(result, str)
+        assert TEST_JSON == my_git.get()
 
 
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_json(gitlab):
     """Check _json_ method."""
     my_git = FileLookup()
 
-    my_git.server.getfile.return_value = {'content': base64.b64encode(TEST_JSON_BYTES)}
-
-    result = my_git.json()
-    assert isinstance(result, dict)
-    assert result['ship'] == 'pirate'
+    with mock.patch.object(FileLookup, 'get', return_value=TEST_JSON) as mock_method:
+        result = my_git.json()
+        assert isinstance(result, dict)
+        assert result['ship'] == 'pirate'
 
 
 @mock.patch('foremast.utils.lookups.gitlab')
 def test_invalid_json(gitlab):
     """Check invalid JSON."""
     my_git = FileLookup()
 
-    my_git.server.getfile.return_value = {'content': base64.b64encode(TEST_JSON_BYTES + b'}')}
-
-    with pytest.raises(SystemExit):
-        my_git.json()
+    with mock.patch.object(FileLookup, 'get', return_value=TEST_JSON + '}') as mock_method:
+        with pytest.raises(SystemExit):
+            my_git.json()
 
 
-@mock.patch('foremast.utils.lookups.gitlab')
-def test_runway_get(gitlab):
+def test_filelookup_attr():
     """Make sure Git related attributes are missing when runway is specified."""
     my_git = FileLookup(runway_dir='/poop_deck')
 
-    assert getattr(my_git, 'git_short') is None
-    assert getattr(my_git, 'server') is None
-    assert getattr(my_git, 'project_id') is None
+    assert my_git.git_short == ''
+    assert my_git.server is None
+    assert my_git.project_id == ''
```

Comment:
`project_id` should not exist. We now care about `project`.

Reply:
Removed.

Reply:
Did not want it removed, should be verifying `self.project` is set to something expected.